quickstart.md

```diff
@@ -0,0 +1,94 @@
+# Quickstart: Gateway
+
+This guide will walk you through running iroh cloud on your local machine, using the default configuration of a gateway backed by a p2p and store service. Once a gateway is up & running we’ll use it to fetch content from the IPFS network by requesting content from a web browser.
+
+
+⚠️ *The quickstart script currently only works for macOS & linux*
+
+
+## 1. Run Iroh locally
+
+we’ve prepped a bash file one-liner for you:
+
+```bash
+$ curl -fsSL http://sh.iroh.computer/setup.sh | sh -s -- quickstart
+```
+
+Running stuff from the internet directly on your terminal is generally a bad idea. Feel free to read through the script first 😄 by dropping the pipe `|` character and everything after it to print `[setup.sh](http://setup.sh)` to your terminal. The setup script takes a bit of time to fetch precompiled binaries. Once up and running you should see terminal output that looks like this:
```

Comment:
https://setup.sh is not the correct url?

Reply:
Also `https` not `http`